rclc_examples/src/example_parameter_server.c

```diff
@@ -144,6 +144,7 @@ int main()
   // clean up
   rc = rclc_executor_fini(&executor);
   rc += rclc_parameter_server_fini(&param_server, &node);
+  rc += rcl_logging_rosout_fini_publisher_for_node(&my_node);
```

Comment:
```suggestion
  rc += rcl_logging_rosout_fini_publisher_for_node(&node);
```

Maybe you have noticed this typo.

Reply:
addressed with https://github.com/ros2/rclc/pull/392/commits/d4e6317a7813ab5f811624709d6e0fe1f67206e2

`colcon test --event-handlers console_direct+ --abort-on-error --packages-select rclc rclc_examples rclc_lifecycle rclc_parameter` 100% tests passed.